### PR TITLE
Made it compatible with the new composition api

### PR DESCRIPTION
**Pull request by Bjornar97**
Exports the functions loadScript and unloadScript, so they can be used directly by importing them. The previous behavior of using this.loadScript still works.

Feel free to make any changes you see fit.

### CODE REVIEW

**Comment by Bjornar97** [comment]
The reason for exporting the functions btw, is that "this" is not available inside the setup function in components.

**Comment by Bjornar97** [comment]
Hi. Is there an update on getting this merged available?

File: index.js [hunk 1]
```diff
@@ -1,6 +1,4 @@
-const LoadScript = {
-  install: function (app) {
-    app.config.globalProperties.$loadScript = function (src) {
+function loadScript(src) {
```
Comment (tserkov) [inline index.js:1]: It looks like your editor's formatter went a little crazy -- please fix the indentation.

File: index.js [hunk 2]
```diff
@@ -27,22 +25,29 @@ const LoadScript = {
...
-        resolve();
-      });
-    };
+    resolve();
+  });
+}
+
+export { unloadScript, loadScript };
+
+const thisLoadScript = {
```
Comment (tserkov) [inline index.js:46]: I think `plugin` would work better than `thisLoadScript`.

**Review by tserkov — CHANGES_REQUESTED** [review]
Hey @Bjornar97! Sorry for _incredible_ delay! I've made a few minor requests to the code, and after they're made I'll smash that merge button.

Thanks for taking care of Composition API support!

**Comment by tserkov** [comment]
Lgtm, thanks again!

**Pull request merged**